frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -64,17 +72,25 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
           <SimpleList isControlled={false} className="ocs-getting-started-card__list">
             {links.map((link) =>
               link.loading ? (
-                <li key={link.key}>
+                <li key={link.id}>
                   <Skeleton fontSize="sm" />
                 </li>
               ) : (
                 <SimpleListItem
-                  key={link.key}
+                  key={link.id}
                   component={link.href ? (link.external ? 'a' : (Link as any)) : 'button'}
                   componentProps={
                     link.external
-                      ? { href: link.href, target: '_blank', rel: 'noopener noreferrer' }
-                      : { to: link.href }
+                      ? {
+                          href: link.href,
+                          target: '_blank',
+                          rel: 'noopener noreferrer',
+                          'data-test-id': `ocs-getting-started-card-item-${link.id}`,
```

Comment:
Use space separated values:
```suggestion
                          'data-test-id': `ocs-getting-started-card-item ${link.id}`,
```